src/charm.py

```diff
@@ -126,6 +129,11 @@ def _pebble_layer(self) -> pebble.LayerDict:
                     "command": "/var/ossec/bin/wazuh-control start",
                     "startup": "enabled",
                     "on-success": "ignore",
+                    "environment": {
```

Comment:
I'm not sure how those env variables are used here but watch out, `""` and `None` can have a different meaning depending on how those envs are consumed. Ref: https://github.com/canonical/loki-k8s-operator/blob/main/lib/charms/loki_k8s/v1/loki_push_api.py#L2147